.github/workflows/workflow.yml

```diff
@@ -31,5 +31,5 @@ jobs:
       - uses: akhileshns/heroku-deploy@v3.4.6
         with:
           heroku_api_key: ${{secrets.HEROKU_API_KEY}}
-          heroku_app_name: "rocketden"
-          heroku_email: "zpchris@wharton.upenn.edu"
+          heroku_app_name: "codejoust"
+          heroku_email: "support@codejoust.co"
```

Comment:
Yeah I wouldn't think so, but this is also untested (since we haven't merged it into main yet) so it's possible an error could come up. But yeah leaning towards no